game-runtimes/unity/unity.mdx

```diff
@@ -34,23 +39,23 @@ If you encounter any errors or unexpected crashes while integrating the Rive Uni
 You can find more details on where to find your Editor.log file in the [Unity docs](https://docs.unity3d.com/Manual/LogFiles.html).
 
 <Note>
- Note that it is best to grab the Editor.log file immediately after a crash has occurred
+  Note that it is best to grab the Editor.log file immediately after a crash has occurred
 </Note>
-## Feature Support 
-
-The rive-unity runtime uses the latest Rive C++ runtime. For more details on runtime support, see the [Feature Support](/feature-support) page. Refer to the following table for what is currently supported in the Unity runtime.
-
-| **Feature** | **Supported** |
-| --- | --- |
-| [Animation Playback](/runtimes/animation-playback) | ✅ |
-| [Fit & Alignment​](/runtimes/layout#fit) | ✅ |
-| [Listeners​](./listeners) | ✅ |
-| [Setting State Machine Inputs​](./state-machines) | ✅ |
-| [Listening to Events](./rive-events) | ✅ |
-| [Updating text at runtime](/runtimes/text) | ✅ |
-| [Out-of-band assets](./loading-assets) | ✅ |
-| [Procedural rendering](./procedural-rendering) | ✅ |
-| PNG images | ✅ |
-| JPEG images | ✅ |
-| WEBP images | ✅ |
 
+## Feature Support
```

Comment:
@damzobridge maybe we should link to the feature support page at some point? Same for Unreal @Tod-Rive .

I want to avoid these tables becoming out of date with new features compared to the Feature Support page.